domains/dscbmr.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "RAYXSTYDIOZ",
+    "email": "syedyaseeralirayan@gmail.com", 
+    "discord":  "1130109258017550357"
+  },
+  "record": {
+    "TXT": "dh=af0561dc7d2f7eb0ef153cb3a270c0b98fe5c36e"
```

Comment:
This is not the right way to connect your domain to Discord, please check here on how to do that https://www.is-a.dev/docs/discord_verification/ (also prior to doing that you need to own dscbmr.is-a.dev)

Reply:
THANK YOU I TRIED DOING IT AGAIN WITH MY DIFFRENT ACCOUNT , SORRY FOR WASTING YOUR TIME